packages/angular_devkit/build_webpack_nodejs/src/binary/index.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * @license
+ * Copyright Google Inc. All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { BuildEvent, Builder, BuilderConfiguration,
+   BuilderContext, BuilderDescription } from '@angular-devkit/architect';
+import { getSystemPath } from '@angular-devkit/core';
+import { ChildProcess, fork } from 'child_process';
+import { resolve } from 'path';
+import { Observable } from 'rxjs';
+import { concatMap, map, switchMap, tap } from 'rxjs/operators';
+import * as treeKill from 'tree-kill';
+import { NodejsBuildBuilderOptions } from '../build';
+import { makeOutfileName } from '../utils/make-outfile-name';
+
+export interface BinaryBuilderOptions {
+  inspect: boolean;
+  args: string[];
+  buildTarget: string;
+  verbose: boolean;
+}
+
+export class BinaryBuilder implements Builder<BinaryBuilderOptions> {
+
+  constructor(public context: BuilderContext) { }
+
+  run(target: BuilderConfiguration<BinaryBuilderOptions>): Observable<BuildEvent> {
+    const root = getSystemPath(this.context.workspace.root);
+    const options = target.options;
+    let subProcess: ChildProcess|null = null;
+
+    // TODO: probably need handling of this ChildProcess
+    const killProcess: () => Observable<void> = () => {
+      return new Observable(obs => {
+        if (!subProcess) {
+          obs.next();
+          obs.complete();
+        } else {
+          treeKill(subProcess.pid, 'SIGTERM', (err) => {
+            if (err) {
+              obs.error(err);
+            }
+            obs.next();
+            obs.complete();
+        });
+        }
+      });
+    };
+
+    const runProcess = (outfile: string) => {
+      return new Observable(obs => {
+        subProcess = fork(outfile, [], {
+          // TODO: should be configurable to inspecet-brk
```

Comment:
There should be a config option to use brk instead of only inspect, but not sure on API.